src/components/Input/TextArea.jsx

```diff
@@ -73,17 +67,19 @@ export const TextArea = ({
         ref={customRef}
         {...rest}
       />
-      {invalid && <ErrorMessage> {errorMsg} </ErrorMessage>}
-      {isLengthExceeded && (
+
+      {isLengthExceeded ? (
         <ErrorMessage>
-          {' '}
           Sorry! It looks like your answer is{' '}
           {maxLength
-            ? `${value.length - maxLength}  character(s)`
+            ? `${value.length - maxLength} character(s)`
             : `${getWords() - maxWords} word(s)`}{' '}
-          over the limit.{' '}
+          over the limit.
         </ErrorMessage>
+      ) : (
+        invalid && <ErrorMessage>{errorMsg}</ErrorMessage>
       )}
+
```

Comment:
these two error messages are basically the same, but kept them both in to be safe. when the length is exceeded, the more descriptive error message is shown, and then when the response is invalid (due to validation for hacker app + other factors in other areas), then it shows the error message that was passed in